c2corg_api/models/mailinglist.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Column, ForeignKey, String
+from sqlalchemy.orm import relationship
+
+from c2corg_api.models import Base, sympa_schema, users_schema
+from c2corg_api.models.user import User
+
+
+class Mailinglist(Base):
+    """
+    Class containing mailing list subscriptions.
+    Based upon Sympa mailing list server https://www.sympa.org/
+    """
+    __tablename__ = 'subscriber_table'
+    __table_args__ = {"schema": sympa_schema}
+
+    listname = Column('list_subscriber', String(50), nullable=False)
+    email = Column(
+        'user_subscriber', String(200),
+        ForeignKey(users_schema + '.user.email'), nullable=False)
```

Comment:
Right, we must propagate any email change in the user account table to the "mailinglist" table. Using a trigger seems a good idea. I have added it to the todo list at https://github.com/c2corg/v6_api/issues/426

By the way ``list_subscriber`` and ``user_subscriber`` are the fields expected by Sympa but I have set aliases (``listname`` and ``email``) to make them more explicit.

Let me recap:
* I should use a ``user_id`` attribute to this ``subscriber_table``, make it a foreign key on the users table
* I keep the ``email`` attribute but it is no longer the foreign key
* I had a trigger to update this ``email`` attribute when the column ``email`` in the users table changes.

Is my ``user`` relationship correct? Should I set it explicitely when creating a ``Mailinglist`` object? As in https://github.com/c2corg/v6_api/blob/dc24a7e486a72e3b99d099125e47f96212cc026f/c2corg_api/tests/views/test_user_mailinglists.py#L18